.swiftlint.yml

```diff
@@ -7,6 +7,7 @@ excluded:
   - IntegrationTests
   - StoreKitTests
   - vendor
+  - scan_derived_data
```

Comment:
this folder is created by `fastlane scan` for `storeKitTests`, and wasn't being excluded so we were getting the same issue as with the `vendor` folder.